### PR TITLE
MAINT: Move to setup.cfg + pyproject.toml

### LINKED ISSUE #95: MAINT: Change installation to ``setup.cfg`` for pip > 19
Comes from #93

### LINKED ISSUE #97: pytests job takes 12min to setup, 4s to run
We should probably not require the Docker image to run pytests.

### PR DESCRIPTION
**Pull request by effigies**
Closes #95.

Also closes #97.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,88 +1,23 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
-# @Author: oesteban
-# @Date:   2015-11-19 16:44:27
-""" smriprep setup script """
+import sys
+from setuptools import setup
+from setuptools.extension import Extension
```
Comment (oesteban) [inline setup.py]: Unused import (as per Codacy)

```suggestion
```

File: smriprep/workflows/norm.py [hunk 2]
```diff
@@ -239,7 +239,7 @@ def init_anat_norm_wf(
     norm_rpt.inputs.after_label = 'Participant'  # after
 
     ds_t1_2_tpl_report = pe.Node(
-        DerivativesDataSink(base_directory=reportlets_dir, keep_dtype=True),
+        DerivativesDataSink(base_directory=reportlets_dir, suffix='T1w'),
```
Comment (oesteban) [inline smriprep/workflows/norm.py]: Do you want to try the old `keep_dtype`?

Comment (effigies) [reply smriprep/workflows/norm.py]: I guess I'd rather make that a separate PR, just in case it's a rabbit hole.

**Comment by effigies** [comment]
Rebased to make reviewing individual commits easier, if needed.

**Comment by effigies** [comment]
@oesteban Want to have another look, with the smaller diff, or just go for it? Tests are passing.

**Comment by oesteban** [comment]
Go for it! :+1:

**Pull request merged**